api/CacheStorage.json

```diff
@@ -49,6 +49,9 @@
               "Accessible from <code>Window</code> from version 43.",
               "Accessible from <code>WorkerGlobalScope</code> from version 43."
             ]
+          },
+          "samsunginternet_android": {
+            "version_added": true
```

Comment:
Tempted to put the version numbers in this file since it landed in 4.0 if i recall correctly.